### PR TITLE
Support RTL accessibility gestures

### PR DESCRIPTION
**Pull request by ASalavei**
Fixes https://youtrack.jetbrains.com/issue/CMP-6875/Support-RTL-in-Accessibility

### Features - iOS
- Support for accessibility gestures for left-to-right languages

### CODE REVIEW

**Review by mazunin-v-jb — APPROVED** [review]

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt [hunk 1]
```diff
@@ -472,18 +477,33 @@ private class AccessibilityElement(
...
-        } else {
-            direction
+
+            UIAccessibilityScrollDirectionDown -> if (axisRange?.reverseScrolling == true) {
+                UIAccessibilityScrollDirectionUp
+            } else {
+                UIAccessibilityScrollDirectionDown
+            }
+
+            UIAccessibilityScrollDirectionRight -> if (isRTL) {
```
Comment (elijah-semyonov) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt]: It's a bit unclear from the context, but should for some reason both `axisRange?.reverseScrolling` _and_ `isRTL` influence the result?

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt [hunk 2]
```diff
@@ -413,47 +414,51 @@ private class AccessibilityElement(
...
-                unclippedRect.right - scrollableAncestorRect.right + scrollableAncestor.size.width / 2,
+                (unclippedRect.right - scrollableAncestorRect.right +
+                    (scrollableAncestor.size.width - unclippedRect.size.width) / 2).invertIfNeeded(),
                 0f
             )
         }
     }
 
-    private fun scrollByIfPossible(dx: Float, dy: Float) {
+    private fun scrollByIfPossible(dx: Float, dy: Float): Boolean {
```
Comment (elijah-semyonov) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt]: Why do we need a `Boolean` added?

**Review by elijah-semyonov — APPROVED** [review]
No blockers, two unclear spots

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt (hunk 2 above)
Comment (ASalavei) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt]: Nice catch. Removed.

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt (hunk 1 above)
Comment (ASalavei) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt]: Thanks, fixed

**Pull request merged**